examples/mundialis_example.py

```diff
@@ -1,21 +1,29 @@
-import openeo
-import logging
+#!/usr/bin/env python3
+
 import json
+import logging
+import openeo
+from openeo.rest.imagecollectionclient import ImageCollectionClient
```

Comment:
normally you should not import `ImageCollectionClient` directly, just a simple `import openeo` should cover most needs. You typically create a "cube" through `connection.load_collection`